.github/workflows/benchmark_execution.yml

```diff
@@ -139,6 +170,14 @@ jobs:
           echo "normal-benchmark-tools-dir=${BENCHMARK_TOOLS_DIR}/build/tools" >> "${GITHUB_OUTPUT}"
           echo "traced-benchmark-tools-dir=${BENCHMARK_TOOLS_DIR}/build-traced/tools" >> "${GITHUB_OUTPUT}"
           echo "tracy-capture-tool=${BENCHMARK_TOOLS_DIR}/build-traced/tracy-capture" >> "${GITHUB_OUTPUT}"
+      - name: "Determine Shard Suffix"
+        id: sharding
+        run: |
+          if [[ ${SHARD_COUNT} -gt 1 ]]; then
```

Comment:
Maybe `if (( SHARD_COUNT > 1 )); then` is shorter (https://google.github.io/styleguide/shellguide.html#arithmetic)